src/rc_allocator.c

```diff
@@ -351,7 +351,7 @@ rc_allocator_init(rc_allocator      *al,
       return rc;
    }
    // To ensure alignment always allocate in multiples of page size.
-   uint32 buffer_size = cfg->extent_capacity * sizeof(uint8);
+   uint64 buffer_size = cfg->extent_capacity * sizeof(uint8);
```

Comment:
Potential bugfix. `buffer_size` declared as uint32 could overflow, to 0, for very large `extent_capacity`. (Discovered this while trying to build negative test cases in newly added, incomplete, `clockcache_test.c` unit test.